maquetado/index.html

```diff
@@ -1,14 +1,24 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
   <link rel="stylesheet" type="text/css" href="application.css">
 </head>
-<body>
-  <h1 class="title">Hello World</h1> 
+
+<body class="all-screen">
+  <div class="row full-width horizontally-to-end" style="height: 328px; max-height: 100%;">
```

Comment:
don't use inline styles, use classes. Inline styles have more specificity than classes and are harder to override.
https://dev.to/emmawedekind/css-specificity-1kca

Reply:
there is no need to set the height or max-height